### PR TITLE
Add postgresql

### PR DESCRIPTION
**Pull request by ianhoffman**
Adds postgres (PSQL) support. To be more specific:
* Creates two tables:
  * `users` stores a user_id and a phone number.
  * `requests` stores a user_id, zipcode, the number of requests from that user for that zipcode, and the timestamps of the first request and last requests by that user for that zipcode.
* Installs `alembic`, a tool used to keep a database in-sync with application code. This way, when we change a model definition in code, the changes will be reflected in the database. For example, if I add a new field to the `User` model, alembic will create the corresponding column in the `users` table.
* Install `flask-migrate`, a tool for connecting alembic to Flask.
* Factors all config stuff into a new module, `settings.py`.
* All postgres stuff lives in `db.py`, including model definitions (just to be clear: "model" == "table", for our intents and purposes. As a model is an object-oriented table definition.)

I foresee launching this to prod involving a hiccup or two, but it hopefully shouldn't be too bad.

### CODE REVIEW

File: app/airq/__init__.py [hunk 1]
```diff
@@ -54,9 +19,10 @@ def healthcheck() -> str:
 @app.route("/sms", methods=["POST"])
 def sms_reply() -> str:
     resp = MessagingResponse()
-    body = request.values.get("Body", "").strip()
-    parts = body.split()
```
Comment (ianhoffman) [inline app/airq/__init__.py:58]: This was dead code.

File: app/airq/db.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    count = db.Column(db.Integer, nullable=False)
+    first_ts = db.Column(db.Integer, nullable=False)
+    last_ts = db.Column(db.Integer, nullable=False)
+
+    def __repr__(self) -> str:
+        return f"<Request {self.zipcode}>"
+
+
+def insert_request(phone_number: str, zipcode: str):
+    if not geodb.get_zipcode_raw(zipcode):
```
Comment (ianhoffman) [inline app/airq/db.py:32]: If this isn't a valid zipcode, just bail out.

File: app/entrypoint.sh [hunk 3]
```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
```
Comment (ianhoffman) [inline app/entrypoint.sh:1]: This runs before the app starts and ensure that the app and DB are in sync.

File: app/migrations/README [hunk 4]
```diff
@@ -0,0 +1 @@
+Generic single-database configuration.
```
Comment (ianhoffman) [inline app/migrations/README:1]: All the code in the `migrations` directory is auto-generated by alembic and executed by running `flask db upgrade`.

**Review by wdanfort — DISMISSED** [review]
impressive!

**Review by wdanfort — APPROVED** [review]

**Pull request merged**